.travis.yml

```diff
@@ -9,13 +9,14 @@ jobs:
 
     - stage: test
       name: Test with PHP 5.6
-      before_script: docker-compose build --build-arg PHP_VERSION=5.6 --build-arg XDEBUG_VERSION=2.5.5 wordpress
+      env:
+        WORDPRESS_IMAGE_VERSION: php5.6
```

Comment:
`docker-compose.yml` now uses the `WORDPRESS_IMAGE_VERSION` environment variable for determining the version of docker image to use.